homeassistant/components/netatmo/sensor.py

```diff
@@ -221,37 +246,31 @@ class NetatmoSensorEntityDescription(SensorEntityDescription):
     ),
     NetatmoSensorEntityDescription(
         key="reachable",
-        name="Reachability",
         netatmo_name="reachable",
         entity_registry_enabled_default=False,
         entity_category=EntityCategory.DIAGNOSTIC,
-        icon="mdi:signal",
     ),
     NetatmoSensorEntityDescription(
         key="rf_status",
-        name="Radio",
         netatmo_name="rf_strength",
         entity_registry_enabled_default=False,
         entity_category=EntityCategory.DIAGNOSTIC,
-        icon="mdi:signal",
+        value_fn=process_rf,
     ),
     NetatmoSensorEntityDescription(
         key="wifi_status",
-        name="Wifi",
         netatmo_name="wifi_strength",
         entity_registry_enabled_default=False,
         entity_category=EntityCategory.DIAGNOSTIC,
-        icon="mdi:wifi",
+        value_fn=process_wifi,
     ),
     NetatmoSensorEntityDescription(
         key="health_idx",
-        name="Health",
         netatmo_name="health_idx",
-        icon="mdi:cloud",
+        value_fn=process_health,
```

Comment:
We should consider deprecating these entities and providing the raw values instead

Reply:
Why? That is exactly what the API has documented.

Reply:
I mean for WiFi and RF we do the conversion ourselves and make it "Fine", "Good" whatever more, instead we could just provide a signal strength sensor that will show the raw value.

(And maybe if that's a bit less user friendly, a lot of other integrations also provide it this way, so this is a global issue to solve imo)